configs/sabaki/gtp-adv600-victim1.cfg

```diff
@@ -0,0 +1,9 @@
+@include gtp-v1.cfg
+
+numBots = 2
+maxVisits0 = 600
+searchAlgorithm0 = EMCTS1
+maxVisits1 = 1
+searchAlgorithm1 = MCTS
```

Comment:
do we need `passingBehavior1 = avoid-pass-alive-territory`? (Depends on whether EMCTS invokes the victim's `Search::getPlaySelectionValues()` on victim nodes, I'm not familiar enough with the EMCTS code)

Reply:
We copy the config in https://github.com/AlignmentResearch/go_attack/blob/main/configs/gtp-amcts.cfg now, since this config was the one used for the 10 million visit benchmark and it worked pretty well.